opencog/atoms/base/Handle.cc

```diff
@@ -216,6 +220,10 @@ std::string oc_to_string(Type type)
 {
 	return atomtype_to_string(type);
 }
+std::string oc_to_string(const LinkPtr& gl)
+{
+	return lptr_to_string(gl);
```

Comment:
Well, that's probably over cautious, some versions of GDB have problems with overloading so when that happens you can still call the non-overloaded version.